<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>921</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbPlaylist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Playlists</string>
     </property>
     <widget class="QPushButton" name="pbAdd">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputPlaylistName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbMusics">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>491</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>Musics</string>
     </property>
     <widget class="QPushButton" name="pbRemoveMusic">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Música</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeViewPlaylist">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>311</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbRemove">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Remove Playlist</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPlay">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPause">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pausar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPlaylistSave">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPlaylisttLoad">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>380</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Carregar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="SearchMusic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>391</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Procurar música</string>
     </property>
     <widget class="QPushButton" name="pbSearch">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leMusicSearchName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Músicas encontradas</string>
      </property>
     </widget>
     <widget class="QTableView" name="twMusicSearchResults">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::TargetMoveAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbAddMusic">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpotify">
    <property name="title">
     <string>Meu Spotify</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConectar"/>
   </widget>
   <addaction name="menuSpotify"/>
  </widget>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
